scipy/stats/_stats_py.py

```diff
@@ -8098,7 +8167,7 @@ def ks_2samp(data1, data2, alternative='two-sided', method='auto'):
             prob = np.exp(expt)
 
     prob = np.clip(prob, 0, 1)
-    return KstestResult(d, prob)
+    return KstestResult(d, prob, statistic_location=d_location, statistic_sign=d_sign)
```

Comment:
```suggestion
    return KstestResult(d, prob, statistic_location=d_location,
                        statistic_sign=d_sign)
```
PEP8: `scipy/stats/_stats_py.py:8170:80: E501 line too long (86 > 79 characters)`